pkgs/tools/security/magnifier/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, llvmPackages
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "magnifier";
+  version = "unstable-2022-08-26";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "magnifier";
+    rev = "27c849db162e509aa413bdd971d7c55e32bc00a3";
+    sha256 = "sha256-yL71f3j9R52c1rFAsB9jPUoBASwdpyFeLSvSAAn9xms=";
+  };
+
+  patches = [ ./no-vendored-llvm.patch ];
```

Comment:
```suggestion
  };
  
  outputs = [ "out" "dev" ];

  patches = [ ./no-vendored-llvm.patch ];
```

Reply:
Basically, `dev` is for includes and libraries to link against, and `out` is for everything else. It should split them out automaticall IIRC.

Reply:
sounds good, applied